src/fastboot-request.js

```diff
@@ -4,7 +4,7 @@ var FastBootHeaders = require('./fastboot-headers');
 function FastBootRequest(request, hostWhitelist) {
   this.hostWhitelist = hostWhitelist;
 
-  this.protocol = request.protocol;
+  this.protocol = request.protocol+':';
```

Comment:
I assume so. Better to drop using string concatenation and instead use template liberals in ES6